core/Stream/hsStdioStream.h

```diff
@@ -28,7 +28,6 @@ class PLASMA_DLL hsStdioStream : public hsFileStream {
     explicit hsStdioStream(bool useStdErr = false);
 
     virtual bool open(const char*, FileMode) { return false; }
-    virtual void close() { }
```

Comment:
This removal is incorrect.  The base class implements close() to close the associated file, which this overrides to remove.